static/app/components/nav/issueViews/issueViewNavItemContent.tsx

```diff
@@ -184,6 +186,7 @@ export function IssueViewNavItemContent({
           if (isInteractingRef.current) {
             e.preventDefault();
           } else {
+            updateViewLastVisited({viewId: view.id});
```

Comment:
We want to count a view as visited regardless of how the user got there, so I'd prefer to do this in a useEffect on the issue stream component. We don't want it tied to the navigation because you might be visiting a view you don't have starred